nanoFramework.CoreLibrary/System/Runtime/CompilerServices/ExcludeFromStubsAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// When a class, enum, or struct is marked with <see cref="ExcludeFromStubsAttribute"/> it will be excluded from the native stubs generated for the nanoFramework interpreter.
```

Comment:
```suggestion
    /// When a class, enum, or struct is marked with <see cref="ExcludeFromStubsAttribute"/> it will be excluded from the native stubs generated for the nanoFramework interpreter.
```
```suggestion
    /// When a class, enum, or struct is marked with <see cref="ExcludeFromStubsAttribute"/> it will be excluded from the native stubs generated by .NET nanoFramework metadata processor.
```